GTG/gtk/errorhandler.py

```diff
@@ -24,25 +24,23 @@ def __init__(self, exception=None, main_msg=None, ignorable: bool = False, conte
 
         formatting = {
             'url': info.REPORT_BUG_URL,
-            'urlname': info.REPORT_BUG_URL,
-            'appname': info.NAME,
+            'urlname': _("our issue tracker"),
         }
 
-        desc2 = _("""Please report the error with the following details and steps to trigger it to:
-<a href="{url}">{urlname}</a>
-Warning: Exiting may lose unsaved changes.""")
-        desc2 = desc2.format(**formatting)
-
         if ignorable:
-            title = _("{appname} encountered an error")
-            desc = _("""{appname} encountered an internal error, but it'll continue running.
-However, more unexpected things can happen, and thus be careful.""")
+            title = _("Internal error")
```

Comment:
I would have included the appname at least in the title so that you can see which application created this window without reading the content.

Reply:
Would appending " — GTG" at the end of those titles do the trick?

Reply:
Yes